dev_guide/builds.adoc

```diff
@@ -549,189 +615,489 @@ source:
 
 // end::using-a-proxy-for-git-cloning-1[]
 
-[[starting-a-build]]
 
-== Starting a Build
-Manually invoke a build using the following command:
+[#using-private-repositories-for-builds]
```

Comment:
I'm not sure this is correct syntax.  I think it should be two-open-square-braces followed by id followed by two-close-square-braces.